components/Templates/class-pods_templates.php

```diff
@@ -221,6 +221,15 @@ public function add_metaboxes( $slug, $post = false ) {
 				'groups' => array(),
 			)
 		);
+		add_meta_box(
+			'pod_reference', __( 'Magic Tag Reference', 'pods' ), array(
+				$this,
+				'render_metaboxes_custom'
```

Comment:
* :no_entry_sign: Comma required after last value in array declaration
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: fa322500090b9ad482beacd9020a39fa</sub>